druid-shell/src/backend/wayland/mod.rs

```diff
@@ -22,7 +22,6 @@ mod events;
 pub mod keyboard;
 pub mod menu;
 pub mod pointers;
-pub mod screen;
```

Comment:
wait you can't just remove screen, druid shell depends on this and will not compile

Reply:
except I did, and it does.... =)

Reply:
ahh i lied. I was building on a different branch. will readd the method. =(

Reply:
I think it is fine to leave it as stub.

Reply:
fixed up and pulled in 1 more small change that fixes an issue with the global id truly being a constant. (w/ a value of 1)